src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -210,7 +210,7 @@ The table below provides descriptions for all of the global decisions that are a
       </td>
 
       <td>
-        Correlation is activated because the New Relic [condition IDs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/violation-event-attributes/#attributes) are the same. For example, cpu usage increase with related services will trigger incidents from the same cpu usage condition, and thus be identified. This logic is valuable beyond [alert policy issue creation preference option](/docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents#preference-options) for one issue per condition, due to condition-level granularity and flexibity in defining correlation time window.
+        Correlation is activated because the New Relic [condition IDs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/incident-event-attributes/#attributes) are the same. For example, cpu usage increase with related services will trigger incidents from the same cpu usage condition, and thus be identified. This logic is valuable beyond [alert policy issue creation preference option](/docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents#preference-options) for one issue per condition, due to condition-level granularity and flexibity in defining correlation time window.
```

Comment:
You should remove docs.newrelic.com from the second URL here

Reply:
Good catch! Thank you!